airflow/contrib/example_dags/example_gcp_transfer.py

```diff
@@ -0,0 +1,276 @@
+                                                   'gcp-transfer-second-target')
+# [END howto_operator_gct_common_variables]
+
+# [START howto_operator_gct_create_job_body_aws]
+create_body_aws = {
+    "description": GCP_DESCRIPTION,
+    "status": "ENABLED",
+    "projectId": GCP_PROJECT_ID,
+    "schedule": {
+        "scheduleStartDate": datetime(2015, 1, 1).date(),
+        "scheduleEndDate": datetime(2020, 1, 1).date(),
+        "startTimeOfDay": datetime.utcnow() + timedelta(minutes=2)
+    },
+    "transferSpec": {
+        'awsS3DataSource': {
+            'bucketName': GCP_TRANSFER_SOURCE_AWS_BUCKET
+        },
+        "gcsDataSink": {
+            "bucketName": GCP_TRANSFER_FIRST_TARGET_BUCKET
+        },
+        "transferOptions": {
+            "overwriteObjectsAlreadyExistingInSink": True
+        },
+    }
+}
+# [END howto_operator_gct_create_job_body_aws]
+
+# [START howto_operator_gct_create_job_body_gcp]
+create_body_gcs = {
+    "description": GCP_DESCRIPTION,
+    "status": "ENABLED",
+    "projectId": GCP_PROJECT_ID,
+    "schedule": {
+        "scheduleStartDate": datetime(2015, 1, 1).date(),
+        "scheduleEndDate": datetime(2020, 1, 1).date(),
+        "startTimeOfDay": datetime.utcnow() + timedelta(minutes=2)
+    },
+    "transferSpec": {
+        'gcsDataSource': {
+            'bucketName': GCP_TRANSFER_FIRST_TARGET_BUCKET
+        },
+        "gcsDataSink": {
+            "bucketName": GCP_TRANSFER_SECOND_TARGET_BUCKET
+        },
+        "transferOptions": {
+            "overwriteObjectsAlreadyExistingInSink": True
+        },
+    }
+}
+# [END howto_operator_gct_create_job_body_gcp]
+
+# [START howto_operator_gct_update_job_body]
+update_body = {
+    "projectId": GCP_PROJECT_ID,
+    "transferJob": {
+        "description": "%s_updated" % GCP_DESCRIPTION
+    },
+    "updateTransferJobFieldMask": "description"
+}
+# [END howto_operator_gct_update_job_body]
+
+list_filter_dict = {
+    "projectId": GCP_PROJECT_ID,
+    "jobNames": []
+}
+
+# [START howto_operator_gct_default_args]
+default_args = {
+    'start_date': days_ago(1)
+}
+# [END howto_operator_gct_default_args]
+
+with models.DAG(
+    'example_gcp_transfer',
+    default_args=default_args,
+    schedule_interval=None  # Override to match your needs
+) as dag:
+
+    def next_dep(task, prev):
+        prev >> task
+        return task
+
+    # [START howto_operator_gct_create_job]
+    create_transfer_job_from_aws = GcpTransferServiceJobCreateOperator(
+        task_id="create_transfer_job_from_aws",
+        body=create_body_aws
+    )
+    # [END howto_operator_gct_create_job]
+
+    prev_task = create_transfer_job_from_aws
+
+    # [START howto_operator_gct_wat_operation]
+    wait_for_operation_to_start = GCPTransferServiceWaitForJobStatusSensor(
+        task_id="wait_for_operation_to_start",
+        job_name="{{task_instance.xcom_pull('create_transfer_from_aws_job', key='return_value')['name']}}",
```

Comment:
There is a problem with this name - it's noot the name of the task: 'create_transfer_from_aws_job' vs. 'create_transfer_job_from_aws'

Reply:
Done